src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/services/GetAddressesForPersonServiceTest.kt

```diff
@@ -53,7 +54,16 @@ internal class GetAddressesForPersonServiceTest(
 
     val result = getAddressesForPersonService.execute(id)
 
-    result.shouldContain(addressesFromProbationOffenderSearch)
-    result.shouldContain(addressesFromNomis)
+    result?.shouldContain(addressesFromProbationOffenderSearch)
+    result?.shouldContain(addressesFromNomis)
+  }
+
+  it("returns null when person cannot be found in all APIs") {
```

Comment:
Wondering whether it's worth qualifying this as 'upstream' apis.

Reply:
I like it, done!